ctapipe/image/cleaning.py

```diff
@@ -111,6 +113,68 @@ def tailcuts_clean(
         )
 
 
+def time_clustering(
+    geom,
+    image,
+    time,
+    pedestal=1.6,
+    n_noise=2.5,
+    minpts=5,
+    eps=1.0,
+    t_scale=4.0,
+    d_scale=0.25,
+):
+    """
+
+    Clean an image by selecting pixels which pass a time clustering algorithm using DBSCAN.
+    Previously used for HESS [timecleaning]_.
+
+    Parameters
+    ----------
+    geom: `ctapipe.instrument.CameraGeometry`
+        Camera geometry information
+    image: array
+        pixel values
+    time: array
+        pixel values
+    pedestal: array
+        pedestal width for each pixel. This is just one number for now.
+    n_noise: array
+        Noise cut
+    minpts: int
+        Minimum number of points to consider a cluster
+    eps: float
+        Distance in dbscan
+    t_scale: float
+        Time scale in ns
+    d_scale: float
```

Comment:
I'd rename these scale factors `time_scale_ns` and `space_scale_m` (or temporal/spatial if you prefer) to be more clear.  Also, might describe why they are there to begin with in the docs (I guess to get the values as close to 1.0 as possible? Does that help the DBSCAN algorithm?).

Reply:
Could also use astropy units here